README.md

```diff
@@ -90,15 +90,21 @@ We request that you cite our following [paper][8] when you use ILLIXR for a publ
 We would also appreciate it if you send us a citation once your work has been published.
 
 ```
-@inproceedings{HuzaifaDesai2021,
-  author={Huzaifa, Muhammad and Desai, Rishi and Grayson, Samuel and Jiang, Xutao and Jing, Ying and Lee, Jae and Lu, Fang and Pang, Yihan and Ravichandran, Joseph and Sinclair, Finn and Tian, Boyuan and Yuan, Hengzhi and Zhang, Jeffrey and Adve, Sarita V.},
-  booktitle={2021 IEEE International Symposium on Workload Characterization (IISWC)}, 
-  title={ILLIXR: Enabling End-to-End Extended Reality Research}, 
-  year={2021},
-  volume={},
-  number={},
-  pages={24-38},
-  doi={10.1109/IISWC53511.2021.00014}
+@ARTICLE {9741292,
```

Comment:
I suggest changing the ID of the Bibtex Entry to `illixr` instead of `9741292`.

Reply:
I have fixed this, and have also added Ubuntu 24 to the install instructions